src/sif/autocost.cc

```diff
@@ -443,7 +446,7 @@ bool AutoCost::AllowedReverse(const baldr::DirectedEdge* edge,
     return false;
   }
 
-  return DynamicCost::EvaluateRestrictions(access_mask_, edge, tile, opp_edgeid, current_time,
+  return DynamicCost::EvaluateRestrictions(access_mask_, edge, false, tile, opp_edgeid, current_time,
```

Comment:
Why is `is_dest` always false for `AllowedReverse` check?

Reply:
What I mean is, if the restriction is near the end and the edges are (likely) processed with `AllowedReverse` during reverse flow, when/how will it be checked? Like we are going from A to K:
```
A---B---C---D---E---F---G---H---I---G---K
                                 ^^^
                             restr here
---->                                <---
fwd                               reverse
```

Reply:
`AllowedReverse` does not check last edge -> `is_dest` always false for `AllowedReverse `.
Check please an update in this PR for `recost_forward` function 
https://github.com/valhalla/valhalla/blob/9c14cf25103bd9ac36df2730bb1a5a842324eb56/src/sif/recost.cc#L97

Reply:
Got it now, ty